spec/rubocop/cop/rspec/example_wording_spec.rb

```diff
@@ -351,5 +351,95 @@
         end
       RUBY
     end
+
+    context "when message includes `'` in `'...'`" do
+      it 'corrects message with `String#inspect`' do
+        expect_offense(<<~'RUBY')
+          it 'should return foo\'s bar' do
+              ^^^^^^^^^^^^^^^^^^^^^^^^ Do not use should when describing your tests.
+          end
+        RUBY
+
+        expect_correction(<<~RUBY)
+          it "returns foo's bar" do
```

Comment:
Maybe it’s a silly question, but why are we changing the type of quotes from `'` to `"` here? If we kept the same quote type, we wouldn’t need to concern ourselves with escaping, would we? (unless _we_ insert an apostrophe/single quote)

Reply:
This is for implementation convenience. If there is an implementation like `string.inspect(quote_type: 'single_quote')`, I would like to use it. If there is a way to rewrite only substrings while preserving quoting and escaping, that would be great, but do you have any other ideas?

Reply:
This is a just idea, but, if the offense range had been like this, we might not have to go through all this trouble?

```
it "should return foo's bar"
    ^^^^^^^^^^^^^
```

Reply:
Fantastic idea!